src/app/components/Headings/index.jsx

```diff
@@ -20,6 +27,10 @@ export const Headline = styled.h1`
 `;
 
 export const SubHeading = styled.h2`
+  color: ${C_STORM};
```

Comment:
I think it might be useful to define styles in the Headline and make this SubHeading just extend the Headline, overriding the padding and colour. Thoughts?

Reply:
https://www.styled-components.com/docs/basics#extending-styles

Reply:
I'd rather have a shared 'base' that both inherit from, since there is a difference in typography as well as padding.

Reply:
Okay fine, but let's not do that. Also, I agree that typography is meant to be different. Currently it's sans reg! https://github.com/bbc/simorgh/pull/403/files#diff-47d43d5e0c8d431009435db20cd0d1abR13